docs/core/tutorials/index.md

```diff
@@ -36,8 +36,13 @@ Choose these tutorials if you want to use Visual Studio Code or some other code
 ## Use Visual Studio for Mac
 
 - [Create a console app](with-visual-studio-mac.md)
-- [Create a library and unit test it](library-with-visual-studio-mac.md)
+- [Debug an app](debugging-with-visual-studio-mac.md)
+- [Publish an app](publishing-with-visual-studio-mac.md)
+- [Create a class library](library-with-visual-studio-mac.md)
+- [Unit test a class library](testing-library-with-visual-studio-mac.md)
 - [Install and use a package](/nuget/quickstart/install-and-use-a-package-in-visual-studio-mac)
+- [Create and publish a package](/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli)
```

Comment:
```suggestion
```

Reply:
@tdykstra Thank you for reviewing!

![image](https://user-images.githubusercontent.com/24927770/92628660-1cff4200-f308-11ea-83ee-c45833341493.png)
There's an issue that redirecting _create-and-publish-a-package-using-the-dotnet-cli_ If click that.

Reply:
Good point. I put it there in the TOC because there isn't a corresponding VS Mac "create and publish" tutorial, so this is the best match we have to offer to make the TOC nodes for VS Mac correspond to the other tools' topics.  On second thought now that you mention it, maybe it would be better to omit the CLI link here too.

Reply:
@tdykstra Thank you for your answer!

Reply:
@tdykstra I updated TOC either, please check it before merging.
Thank you.